src/chrome/content/rules/multistre.am.xml

```diff
@@ -0,0 +1,5 @@
+<ruleset name="multistre.am">
+	<target host="multistre.am" />
```

Comment:
`https://www.multistre.am/` redirects to `http://multistre.am/` and we will redirect it to `https://multistre.am/`. That's totally fine.